src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext } from 'react';
+import { TodoItem } from '../TodoItem';
+import { TodosContext } from '../TodosContext';
+
+type Props = {};
+
+export const TodoList: React.FC<Props> = () => {
```

Comment:
```suggestion
export const TodoList = () => {
```
You don't have any props